app/assets/js/app/board.coffee

```diff
@@ -49,6 +50,12 @@ class kablammo.Board
       ammos.find("li").removeClass('ammo-full').addClass('ammo-empty')
     else
       ammos.find("li:gt(#{robot.ammo-1})").removeClass('ammo-full').addClass('ammo-empty')
+  
+  # Canvas height can't be percentage based.
+  updateCanvasHeight: => 
+    $board = $('#board');
+    console.log($board.width())
```

Comment:
Should we get rid of this `console.log`?

Reply:
Ah shoot, missed that one!